### DISCUSSION TITLE
Forecast no 24h display by default

### DISCUSSION THREAD

**Discussion by f4gqk** [poblabs/weewx-belchertown#792] [Q&A]
Hello,

in the "Forecast" part of my website www.meteo-guignes.fr, I do not have the 24 hour display by default.

How to set it by default?

What is the place where we choose either 1h, 3h, or 24h?

Thanks

Sebastien

**Comment by michaelundwd** [comment]
When I look at your website, 24hr interval appears to be set by default, so am wondering if you have cleared the cache on your browser.  

You will find the instructions in the Belchertown wiki under the Forecast Options heading.  forcast_interval_hours should be set to 24.

**Reply by f4gqk** [reply]
Thanks

it works by clearing the cache

Sébastien
